frontend/sass/norent/_letter-builder.scss

```diff
@@ -1,3 +1,26 @@
+// This class is a recreation of the .jf-above-footer-content class
+// that allows our footer to peak above the fold on desktop and
+// fit just below the fold on mobile, while maintaining the
+// correct formatting of NoRent internal pages
+.jf-norent-internal-above-footer-content {
+  min-height: calc(100vh - #{$navbar-height});
+  @include tablet {
+    margin-top: 4rem;
+    margin-bottom: 4rem;
+    min-height: calc(100vh - 8rem - #{$navbar-height} - #{$footer-offset});
+  }
+}
+
+// Fixes page container spacing issue for users in safe mode
+html[data-safe-mode-no-js] .jf-norent-internal-above-footer-content {
+  @include tablet {
+    margin-top: 0;
+    nav.navbar {
+      margin-bottom: 4rem;
+    }
+  }
+}
```

Comment:
o cool thanks for making sure this works in safe mode!

Reply:
you bet!